foundations/html_css/intro-to-html-css.md

```diff
@@ -18,23 +18,23 @@ Many great resources out there keep referring to HTML and CSS as _programming la
 
 <div class="lesson-content__panel" markdown="1">
 
-1.  Read [HTML vs CSS vs JavaScript](https://brytdesigns.com/html-css-javascript-whats-the-difference/). It's a quick overview of the relationship between HTML, CSS, and JavaScript.
+1. Read [HTML vs CSS vs JavaScript](https://brytdesigns.com/html-css-javascript-whats-the-difference/). It's a quick overview of the relationship between HTML, CSS, and JavaScript.
 
 </div>
 
-### Knowledge Check
+<div class="lesson-content__panel" markdown="1">
 
-This section contains questions for you to check your understanding of this lesson on your own. If you’re having trouble answering a question, click it and review the material it links to.
+1.  Read [HTML vs CSS vs JavaScript](https://brytdesigns.com/html-css-javascript-whats-the-difference/). It's a quick overview of the relationship between HTML, CSS, and JavaScript.
 
--   [What do HTML and CSS stand for?](https://brytdesigns.com/html-css-javascript-whats-the-difference/#What_is_HTML)
--   [Between HTML and CSS, which would you use for putting paragraphs of text on a webpage?](#html-and-css)
--   [Between HTML and CSS, which would you use for changing the font and background color of a button?](#html-and-css)
--   [What is the difference between HTML, CSS and JavaScript?](https://brytdesigns.com/html-css-javascript-whats-the-difference/)
+</div>
 
-### Additional Resources
```

Comment:
Your file looks to have removed the additional resources section.

Reply:
I'm sorry >.< this is my first time contributing, should I start a new pull request and fix all these mistakes?

Reply:
Nevermind, I think I figured out how to tackle them on github. The changes have been committed for all except the requests I specified.